circle-mlir/circle-mlir/tools/onnx2circle/TestModels.cmake

```
@@ -62,6 +62,22 @@ macro(ValidateDynaShapeInf MLIR_FNAME)
   list(APPEND FILE_DEPS "${TEST_MLIR_MODEL_DST}")
 endmacro(ValidateDynaShapeInf)
 
+# ValidateDynaBatchInf is to test dynamic_batch_to_single_batch option
+set(VALIDATE_DYNABATCHTOSINGLE_MODELS)
+macro(ValidateDynaBatchToSingle MLIR_FNAME)
+  # copy to build folder
+  set(TEST_MLIR_MODEL_SRC "${CMAKE_SOURCE_DIR}/models/mlir/${MLIR_FNAME}")
+  set(TEST_MLIR_MODEL_DST "${CMAKE_CURRENT_BINARY_DIR}/models/mlir/${MLIR_FNAME}")
+  add_custom_command(
+    OUTPUT ${TEST_MLIR_MODEL_DST}
```

(Optional)
In `add_custom_command`, the cmake -E copy step will fail if the `${CMAKE_CURRENT_BINARY_DIR}/models/mlir` directory doesn’t exist. It would be better to insert a `make_directory` invocation before the copy, for example:
```suggestion
    OUTPUT ${TEST_MLIR_MODEL_DST} 
    COMMAND ${CMAKE_COMMAND} -E make_directory
        "${CMAKE_CURRENT_BINARY_DIR}/models/mlir"
```

---

I didn't have experience copy failure, and my suggestion was purely preventative. If there is no issue, I think it's fine to leave it as is.